okta/group.go

```diff
@@ -65,9 +65,9 @@ func listGroups(ctx context.Context, client *okta.Client, qp *query.Params) ([]*
 // Group Primary Key Operations (Use when # groups < # users in operations)
 func addGroupMembers(ctx context.Context, client *okta.Client, groupId string, users []string) error {
 	for _, user := range users {
-		resp, err := client.Group.AddUserToGroup(ctx, groupId, user)
-		exists, err := doesResourceExist(resp, err)
-		if err != nil {
+		resp, err1 := client.Group.AddUserToGroup(ctx, groupId, user)
+		exists, err2 := doesResourceExist(resp, err)
+		if err1 != nil || err2 != nil {
 			return fmt.Errorf("failed to add user (%s) to group (%s): %w", user, groupId, err)
```

Comment:
![image](https://user-images.githubusercontent.com/201/189756777-ad3d7064-4e90-4afe-8437-8d1054c31626.png)

I almost accepted this PR blind. I'll do a separate PR for this